### PR TITLE
Add --environ-file for defaults for env object

### PR DESCRIPTION
**Pull request by kanaka**
Use `--environ-file` option that parses and environment file in order to populate the base env object/map.

This uses version 1.24.0 of dotenvx (PR adding necessary functionality: https://github.com/dotenvx/dotenvx/issues/445)

Closely related to https://github.com/LonoCloud/conlink/pull/32

### CODE REVIEW

**Review by jonsmock — CHANGES_REQUESTED** [review]
Following up on the actual PR with out-of-band discussion, looks good to me with the agreed upon changes:

- rename flag to `--environ-file-unsafe` (future can add `--environ-file`) due to node versions prior to node 22 consuming the `--env-file` https://github.com/babashka/nbb/issues/371
- addition of at least one test that exercises it in GHA somehow

File: src/dctest/util.cljs [hunk 1]
```diff
@@ -8,9 +8,10 @@
     [clojure.pprint :refer [pprint print-table]]
     [dctest.expressions :as expr]
     [promesa.core :as P]
-    [viasat.util :refer [Eprintln fatal read-file]]
+    [viasat.util :refer [Eprintln fatal read-file exec-promise]]
```
Comment (jonsmock) [inline src/dctest/util.cljs]: not important: probably don't need exec-promise anymore :-)

**Review by jonsmock — APPROVED** [review]
Yay! Looks awesome!

File: src/dctest/util.cljs (hunk 1 above)
Comment (kanaka) [reply src/dctest/util.cljs]: I fixed that, thanks.

**Pull request merged**